sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuHashJoin.scala

```diff
@@ -128,7 +128,9 @@ object GpuHashJoin {
       case _ =>
         meta.willNotWorkOnGpu(s"$joinType currently is not supported")
     }
+  }
 
+  def tagBuildSide(meta: SparkPlanMeta[_], joinType: JoinType, buildSide: GpuBuildSide): Unit = {
```

Comment:
Can we have some comments on where this tagging will be used? Since I saw two "tagBuildSide" for join things.